server.js

```diff
@@ -19,9 +24,11 @@ app.get('/', function (req, res) {
   res.sendFile(path.join(__dirname + '/dist', 'index.html'));
 });
 
-app.get('/hello', function(req, res) {
+/* app.get('/hello', function(req, res) {
```

Comment:
avoid code block commenting, delete the code if you don't need,  in this case remove the code block.